views/js/controller/runner/runner.js

```diff
@@ -67,15 +68,33 @@ define([
         next        : next,
         skip        : skip,
         nextSection : nextSection,
-        overlay     : overlay
+        overlay     : overlay,
+        dialog      : dialog
     };
 
     /**
      * The runner controller
      */
     var runnerController = {
+
+        /**
+         * Controller entry point
+         *
+         * TODO verify required options
+         *
+         * @param {Object} options - the testRunner options
+         * @param {String} options.testDefinition
+         * @param {String} optins.testCompilation
+         * @param {String} optins.serviceCallId
+         * @param {String} optins.serviceController
+         * @param {String} optins.serviceExtension
+         * @param {String} optins.exitUrl - the full URL where to return at the final end of the test
```

Comment:
Misspelling name of parameter in the doc: `optins` should be replaced by `options`